src/main/java/com/ibm/cloud/sdk/core/security/IamAuthenticator.java

```diff
@@ -421,7 +421,8 @@ public IamToken requestToken() {
     // Form a POST request to retrieve the access token.
     RequestBuilder builder = RequestBuilder.post(RequestBuilder.resolveRequestUrl(this.getURL(), OPERATION_PATH));
 
-    // Now add the Content-Type and (optionally) the Authorization header to the token server request.
+    // Now add the Accept, Content-Type and (optionally) the Authorization header to the token server request.
+    builder.header(HttpHeaders.ACCEPT, HttpMediaType.APPLICATION_JSON);
```

Comment:
I noticed that we weren't previously setting the Accept header on outbound IAM get-token requests, so added that here.  Apparently the IAM server doesn't care, but... :)